packages/core/src/dialects/db2/query-generator.js

```diff
@@ -823,7 +761,7 @@ export class Db2QueryGenerator extends Db2QueryGeneratorTypeScript {
   }
 
   dropForeignKeyQuery(tableName, foreignKey) {
-    return _.template('ALTER TABLE <%= table %> DROP <%= key %>', this._templateSettings)({
+    return _.template('ALTER TABLE <%= table %> DROP FOREIGN KEY <%= key %>', this._templateSettings)({
```

Comment:
Seems that this didn't work at all before? Or I'm missing something, but I think this will be the next QG function I want to unify and improve tests for. This time also with integration tests

Reply:
The documentation does say it's DROP FOREIGN KEY, but DB2 being the dialect that silently muted errors in query.js, I wouldn't be surprised if that's what happened here .-.